src/components/components/ReinforcementStatement/ReinforcementStatement.scss

```diff
@@ -0,0 +1,52 @@
+.Reinforcement {
+  display: flex;
+  align-items: center;
+  margin: 96px;
+  gap: 96px;
```

Comment:
I would suggest making the gap responsive (e.g. gap: 6em ?)
and similar for the margins, or maybe it is better to make a responsive width and auto margins? like e.g. 
 width: 80%;
  margin: 0 auto;
  gap: 5em;

Reply:
Let's ignore this here for now. We can adjust this later I think

Reply:
Ok Thank You!